doc/colvars-refman-main.tex

```diff
@@ -2232,6 +2232,122 @@ \subsection*{Overview}
 \end{cvcoptions}
 
 
+\cvsubsubsec{\texttt{eulerPhi}: Row angle from 
+references coordinates.}{sec:cvc_eulerPhi}
+\labelkey{colvar|eulerPhi}
+
+Assuming the axes of the original frame are denoted as \textit{x, y, z} and the 
```

Comment:
This paragraph would be nicely complemented by a diagram. Maybe the one from Wikipedia, edited for the notation?
https://en.wikipedia.org/wiki/Euler_angles#/media/File:Eulerangles.svg

Reply:
I use `\includegraphics` in colvars-refman-main.tex to include the PDF version of the figure. Is that a problem for generating the webpage manual?

Reply:
It should work, but I can check tomorrow and if it works merge as is.